data/presets/aeroway/runway.json

```diff
@@ -7,11 +7,19 @@
         "width"
     ],
     "geometry": [
-        "line"
+        "line",
+        "area"
     ],
     "tags": {
         "aeroway": "runway"
     },
+    "addTags": {
+        "aeroway": "runway"
+    },
+    "removeTags": {
+        "aeroway": "*",
+        "area:aeroway": "*"
```

Comment:
a simpler solution would be to leave the existing preset untouched, and create a single new file called `runway_area.json`.

[`pedestrian_line.json`](https://github.com/openstreetmap/id-tagging-schema/blob/main/data/presets/highway/pedestrian_line.json) and [`pedestrian_area.json`](https://github.com/openstreetmap/id-tagging-schema/blob/main/data/presets/highway/pedestrian_area.json) is a good example